src/cm.css

```diff
@@ -504,6 +504,25 @@
   }
 }
 
+
+/*******************
+ * DoNotSellExplainer
+ *******************/
+
+/** Styles for button of X marker in top right corner */
+.do-not-sell-explainer-close {
+  position: absolute;
+  top: 12px;
+  right: 12px;
+  border: none;
+  cursor: pointer;
+}
```

Comment:
```suggestion
.do-not-sell-explainer-close {
  position: absolute;
  top: 12px;
  right: 12px;
  border: none;
  cursor: pointer;
  background: none;
}
```

Reply:
Not loving this background with hover 

<img width="82" alt="Screen Shot 2022-12-22 at 8 38 23 AM" src="https://user-images.githubusercontent.com/7354176/209157843-05372fea-6b10-4542-bad1-f66254d458d9.png">